packages/grid/_modules_/grid/hooks/features/keyboard/useKeyboard.ts

```diff
@@ -232,7 +232,7 @@ export const useKeyboard = (gridRootRef: React.RefObject<HTMLDivElement>, apiRef
         return;
       }
 
-      if (!isTabKey(event.code)) {
+      if (!isTabKey(event.code) && !isCell(document.activeElement)) {
```

Comment:
It made me think about https://github.com/mui-org/material-ui-x/issues/132.

Reply:
Can I do something about it ?

Reply:
Not in the pull request, but I think that we can make a pull request to replace all usages of `event.code`, it would match what we do in the main repo and likely rightfully: .

> If you are building a game that uses WASD keys, then use .code, as it will work across any keyboard layout. However, if you are interested in detecting if specific inputs were pressed (e.g. arrows, home, end, page up, page down), then use .key. I believe most applications will end up using .key instead of .code.

https://codepen.io/denilsonsa/pen/epmoma